### PR TITLE
5060 georchestra review

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description
Review of #5128 and #5132  

## Other useful information
RdD [link](https://elena-mapstore-docs.readthedocs.io/en/5060_georchestra_review/user-guide/managing-contexts/)

### CODE REVIEW

File: docs/user-guide/application-context.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Application Context
+*******************
+
```
Comment (mbarto) [inline docs/user-guide/application-context.md:3]: Is there any "introduction" (here  or elsewhere) of what an Application Context is and why an admin should "desire" to create one?

File: docs/user-guide/managing-contexts.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Managing Contexts
+*******************
+
+On [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) the **Application Context Manager** is an administrative tool designed to build and configure MapStore's viewers: the administrator is able to configure a custom MapStore viewer by choosing:
```
Comment (tdipisa) [inline docs/user-guide/managing-contexts.md:4]: In MapStore the Application Context Manager ...

File: docs/user-guide/map-templates.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Map Templates
+*******************
+
+This extension allows to browse **Map Templates** in a MapStore's viewer. Supported Map Templates formats in MapStore are [WMC](https://mapstore.readthedocs.io/en/latest/developer-guide/maps-configuration/#web-map-context) and [MapStore's native JSON](https://mapstore.readthedocs.io/en/latest/developer-guide/maps-configuration/#map-options). The <img src="../img/button/map-templates-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provide to the user the list of the available templates.
```
Comment (tdipisa) [inline docs/user-guide/map-templates.md:4]: ... button, present in *Burger Menu* , **provides** to the user ...

File: docs/user-guide/application-context.md (hunk 1 above)
Comment (tdipisa) [reply docs/user-guide/application-context.md:3]: https://elena-mapstore-docs.readthedocs.io/en/5060_georchestra_review/user-guide/managing-contexts/

**Pull request merged**